### ISSUE TITLE
Add -RunAsAdmin Flag

### ISSUE THREAD

**Issue by MalwareMechanic** [mandiant/VM-Packages#61]
Add option for helper functions to allow user's to specify if tool should be ran as admin

**Comment by Ana06** [comment]
@emtuls is this issue still needed?

**Comment by emtuls** [comment]
I think it should be added as a parameter to a few more functions inside of `common.vm` just so that we have the option available for future packages.

Right now it's only used in `VM-Install-Shortcut`, which means that in order for us to have a tool be run as admin, we have to call the `VM-Install-Shortcut` directly with the `-runAsAdmin` flag.

If we also add this flag to `VM-Install-From-Zip`, `VM-Install-With-Installer`, `VM-Install-Single-Exe`, and `VM-Install-Single-Ps1`, then we would be able to just pass the flag along to the `VM-Install-Shortcut` call so that it is handled as needed.

**Comment by naacbin** [comment]
I agree, this would be a useful addition as currently packages such as `systeminformer` are not run as admin.

**Comment by PrajeetGuha** [comment]
will resolve this as I am also facing issue with **systeminformer** is not working in full capacity without the administrative privilege

**Comment by Ana06** [comment]
`VM-Install-Shortcut` supports a `RunAsAdmin` parameter, but we need to add it to the helper functions (such as `VM-Install-From-Zip` that is the one SystemInformer uses). @PrajeetGuha it would be great if you could address this issue! 🚀

**Comment by Ana06** [comment]
@PrajeetGuha I have just given you triage permissions. I think you should be able to now assign yourself issues (please only do this if the issue is not assigned to someone else already) and add tags as well. 😉
